schema.sql

```diff
@@ -1,13 +1,50 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
+CREATE TABLE Animals (
     id            INT GENERATED BY DEFAULT AS IDENTITY,
     name TEXT NOT NULL, 
     date_of_birth DATE NOT NULL, 
     escape_attempts INT NOT NULL, 
     neutered BOOLEAN NOT NULL, 
     weight_kg DECIMAL NOT NULL,
     PRIMARY KEY(id)
-);
+)
+
+ALter table Animals ADD species TEXT NOT NULL;
+
+CREATE TABLE owners (
+    id            SERIAL PRIMARY KEY,
+    full_name TEXT NOT NULL;
+    age INT NOT NULL; 
+)
+
+CREATE TABLE species (
+    id        SERIAL PRIMARY KEY,
+   name TEXT NOT NULL; 
+)
+
+-- Modify animals table:
+
+--     Make sure that id is set as autoincremented PRIMARY KEY
+
+
+
+BEGIN;
+ALTER TABLE Animals DROP CONSTRAINT "GENERATED BY DEFAULT AS IDENTITY" FROM id;
+ALTER TABLE Animals ADD CONSTRAINT "SERIAL" PRIMARY KEY (id);
+COMMIT;
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/172015439-d3c7bc04-54b3-4f7c-9ad5-1579a8fd3d3f.png)

- [ ] Kindly this code is throwing an error kindly correct it.

Reply:
Thank you for pointing this out. I've removed this line of code. The Constraint "GENERATED BY DEFAULT AS IDENTITY" earlier added while creating the column ID has an implicit sequence that will add and set a value serially if not given.